openmc/settings.py

```diff
@@ -229,6 +229,10 @@ class Settings:
         .. versionadded:: 0.13
     write_initial_source : bool
         Indicate whether to write the initial source distribution to file
+    create_delayed_neutrons : bool
+        Whether delayed neutrons are created in fission.
+        
+        .. versionadded:: 0.13
```

Comment:
Can you change the version to 0.13.3 and move this up in the last of attributes (they are listed alphabetically)?